akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -0,0 +1,93 @@
+# Rolling Updates
+
+A rolling update is the process of replacing one version of the system with another without downtime.
+The changes can be new code, changed dependencies such as new Akka version, or modified configuration.
+
+In Akka, rolling updates are typically used for a stateful Akka Cluster where you can't run two separate clusters in
+parallel during the update, for example in blue green deployments.
+
+For rolling updates related to Akka dependency version upgrades and the migration guides, please see
+@ref:[Rolling Updates and Akka versions](../project/rolling-update.md)
+
+#### This document covers: 
+* [Serialization Compatibility](#serialization-compatibility)
+* [Cluster Sharding](#cluster-sharding)
+* [Migrating Untyped to Typed](#migrating-untyped-to-typed)
+* [Cluster Shutdown](#cluster-shutdown)
+* [Cluster Configuration Compatibility Check](#cluster-configuration-compatibility-check)
+ 
+## Serialization Compatibility
+
+There are two parts of Akka that need careful consideration when performing an rolling update.
+
+1. Compatibility of remote message protocols. Old nodes may send messages to new nodes and vice versa.
+1. Serialization format of persisted events and snapshots. New nodes must be able to read old data, and
+   during the update old nodes must be able to read data stored by new nodes.
+
+There are many more application specific aspects. It's important to have a strategy for serialization
+format that can be evolved and you can find advice in
+@ref:[Persistence - Schema Evolution](../persistence-schema-evolution.md), which also applies to
+remote messages when deploying with rolling updates.
+
+### Retiring a serializer without downtime
+
+One approach is carried out in two phases: 
+
+1. Deploy a version of your application that still supports deserializing messages in the old form 
+(your deserializer is still in "akka.actor.serializers"-section of your configuration, but it no longer serializes new messages 
+(it is removed from “akka.actor.serialization-bindings”-section). See @ref:[Serialization Configuration](../serialization.md#configuration) for details.
+1. After this upgrade has succeeded and there are no longer systems sending messages serialized with the decommissioned serializer, 
+you can upgrade to a version that also no longer contains this serializer in `akka.actor.serializers`.
+
+## Cluster Sharding
+
+During a rolling upgrade, sharded entities receiving traffic may be moved during @ref:[shard rebalancing](../cluster-sharding.md#shard-rebalancing), 
+to an old or new node in the cluster, based on the pluggable allocation strategy and settings.
+When an old node is stopped the shards that were running on it may be allocated to one of the
+other old nodes remaining in the cluster. 
+ 
+It is recommended to leave the oldest node until last, because the shard coordinator 
+is running on that node. See @ref:[Cluster Shutdown](#cluster-shutdown) below for more detail.
```

Comment:
I wonder if 'it is recommended' is too strong here: it definitely can be a useful optimization, but for many systems it shouldn't be an issue?

Reply:
Sure, I got that from something of ours that used the phrase, "it is recommended..." for this ;) Updated.